app/packages/core/src/components/Actions/ActionsRow.tsx

```diff
@@ -162,13 +162,14 @@ const Tag = ({
   const [available, setAvailable] = useState(true);
   const labels = useRecoilValue(fos.selectedLabelIds);
   const samples = useRecoilValue(fos.selectedSamples);
-  const readOnly = useRecoilValue(fos.readOnly);
+  const canTag = useRecoilValue(fos.canTagSamplesOrLabels);
```

Comment:
cool, this should work even after introducing the TAG permission since TAG < EDIT. so no feature flagging for dynamic collaborator is required on teams side.